index.html

```diff
@@ -351,6 +351,8 @@ <h3 class="fallback-page__heading">Your browser is out of date</h3>
       async="async"
       defer="defer"
       data-show-fbm="false"
+      data-app-title="Search"
+      data-app-link="search"
```

Comment:
make this "" otherwise weird things happen

Reply:
I tried `/` and it worked. I also tried `/search` and it worked too.